src/physics/PhysicsSystem.java

```diff
@@ -84,24 +84,30 @@ public void update() {
 				currTrans.increasePosition(newVeloc.mul(super.getDeltaTime()));
 			}
 		}
-		
+
 		super.timeMarkEnd();
 	}
-	
+
 	public void cleanUp() {
-		
+
 	}
-	
+
 	public void loadBlueprint(ArrayList<String> blueprint) {
 		// - PhysicsComponent
 		ArrayList<String> physicsComponentData = BlueprintLoader.getAllLinesWith("PHYSICSCOMPONENT", blueprint);
 		String[] frags = null;
-		for(String dataSet : physicsComponentData) {
-			int eID = BlueprintLoader.extractEID(dataSet);
-			frags = BlueprintLoader.getDataFragments(dataSet);
-			entityController.addPhysicsComponent(eID)
-				.setWeight(Float.valueOf(frags[0]))
-				.setAffectedByGravity(Boolean.valueOf(frags[1]));
+		for (String dataSet : physicsComponentData) {
+			try {
+				int eID = BlueprintLoader.extractEID(dataSet);
+				if (eID < 0) {
+					continue;
+				}
+				frags = BlueprintLoader.getDataFragments(dataSet);
+				entityController.addPhysicsComponent(eID).setWeight(Float.valueOf(frags[0]))
+						.setAffectedByGravity(Boolean.valueOf(frags[1]));
+			} catch (NullPointerException | IllegalArgumentException e) {
```

Comment:
Refer to other comment on an identical line.

Reply:
Also maybe include the eID this component was supposed to belong to make finding and fixing the correct line easier